apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/Components/AppealSection/AppealSection.tsx

```diff
@@ -59,74 +42,79 @@ const AppealSection: React.FC<Props> = (props) => {
           Ákvörðun um kæru
         </Text>
       </Box>
-      {!isAppealDeadlineExpired && (
+      {!workingCase.isAppealDeadlineExpired && workingCase.rulingDate && (
```

Comment:
Do you need to check rulingDate? You should not be on this page unless you have a ruling date.

Reply:
I need to check `rulingDate` because the `rulingDate` parameter in `getAppealEndDate` function is of type string but `rulingDate` is of type `string | undefined` in `Case`

![Screen Shot 2021-06-25 at 15 16 38](https://user-images.githubusercontent.com/3789875/123446386-591a3f80-d5c8-11eb-8bc4-b1f69394ac85.png)